hw/ip/rv_dm/data/rv_dm.hjson

```diff
@@ -282,7 +291,13 @@
       },
       { skipto: "0x108" }
       { name: "GOING",
-        desc: "Write to this address to acknowledge that the core is executing.",
+        desc: '''
+              A hart that receives an abstract command (indicated by its corresponds !!FLAGS register) must write to this address to acknowledge it received the command.
+              The value written is unused, but it is conventionally 0.
+
+              Upon receiving the write, the debug module will reset the GO field in the selected hart's !!FLAGS register.
+              The debug module will transition to a state where it awaits the write to !!HALTED to indicate the command has completed.
```

Comment:
Note that this is not how the DM currently operates, and I think that is a bug (assuming I am following the code correctly).

https://github.com/lowRISC/opentitan/blob/beac109dc22744615664dc13bc728df226237339/hw/vendor/pulp_riscv_dbg/src/dm_mem.sv#L189-L196

The current code is looking at the `halted` value directly, but a hart executing an abstract command is _already_ halted.

Reply:
@a-will 
Are you suggesting we need to update dm_mem.sv to match the behavior description?

Reply:
Yes. It seems like we're not manifesting a bad outcome here (at least... as far as we can tell), and I'm not totally sure why not. However, this looks like a bug we ought to fix.

Reply:
> Yes. It seems like we're not manifesting a bad outcome here (at least... as far as we can tell), and I'm not totally sure why not. However, this looks like a bug we ought to fix.

OK, can you file an issue to trace?

Reply:
Done.